backend/danswer/document_index/vespa/chunk_retrieval.py

```diff
@@ -130,7 +132,7 @@ def _vespa_hit_to_inference_chunk(
         document_id=fields[DOCUMENT_ID],
         source_type=fields[SOURCE_TYPE],
         title=fields.get(TITLE),
-        semantic_identifier=fields[SEMANTIC_IDENTIFIER],
+        semantic_identifier=fields.get(SEMANTIC_IDENTIFIER, ""),
```

Comment:
Why would this ever be empty or missing? This is probably one of those cases we should fail loudly. If a doc has no semantic identifier, we have no way of showing it to the user in the UI. We wouldn't know what to call it. Better to actually understand what is causing this

Reply:
Agreed– I don't think we should merge this PR, but needed a quick hotfix for user who was having a bad time with wonky Vespa instance